README.adoc

```diff
@@ -46,7 +46,7 @@ to directly convert JSON messages into Java objects. The second approach consist
 using the Java API for JSON Processing (JSON-P) to process the JSON.
 
 The REST service that provides the artists and albums resources has already been written
-for you and is accessible at the following link when the server is running http://localhost:9080/artists[http://localhost:9080/artists^], which responds with the [hotspot]`artists.json`.
+for you and is accessible at the following link when the server is running `\http://localhost:9080/artists`, which responds with the [hotspot]`artists.json`.
```

Comment:
suggested edit:
The REST service that provides the artists and albums resources has already been written for you and is accessible at the following link when the server is running http://localhost:9080/artists, which responds with the artists.json.
--->
The REST service that provides the artist and album resources is already written for you. When the server is running, this service  is accessible at the  http://localhost:9080/artists endpoint, which responds with the artists.json file.

if artists.json  is a file? or is it and object?

Reply:
should be file